tasks/module_4/task_7.py

```diff
@@ -0,0 +1,53 @@
+"""Module4/Task7: Excel to CSV converter."""
+import os
+import csv
+
+from openpyxl import load_workbook
+
+from tasks.module_4.task_2 import cwd
+
+
+# Vars
+EXCEL_DIR = 'resources/task_7'
+EXCEL_EXTENSIONS = ['.xlsx']
+
+
+def get_excel_files() -> list:
+    """Get list of excel files in current directory."""
+    files = []
+    for file_ in os.listdir():
+        if os.path.isfile(file_) and os.path.splitext(file_)[1] in EXCEL_EXTENSIONS:
+            files.append(file_)
+    return files
+
+
+def convert_excel_to_csv(file: str):
```

Comment:
Maybe `filename` or `filepath` would be better choice than just `file`. What do you think?